inloop/solutions/validators.py

```diff
@@ -0,0 +1,13 @@
+from constance import config
+
+
+def validate(file_name):
+    file_name = file_name.lower()
+    for ending in parse_allowed_file_endings():
+        if file_name.endswith(ending):
+            return True
+    return False
+
+
+def parse_allowed_file_endings():
+    return [ending.strip().lower() for ending in config.ALLOWED_FILE_ENDINGS.split(",")]
```

Comment:
Just as a side note: accessing slots of the `config` object results in a redis query each and every time.

Reply:
I solved this with calling a signal on config change. What do you think about this? Let me know in a new review. Thanks!